operator/v1/types_network.go

```diff
@@ -234,8 +234,9 @@ type OpenShiftSDNConfig struct {
 	// +optional
 	MTU *uint32 `json:"mtu,omitempty"`
 
-	// useExternalOpenvswitch tells the operator not to install openvswitch, because
-	// it will be provided separately. If set, you must provide it yourself.
+	// useExternalOpenvswitch is ignored. This used to control whether Open vSwitch
+	// was run as a system service or in a container, but as of OpenShift 4.7,
+	// OpenShift always runs it as a system service.
```

Comment:
Mark it as deprecated?

Reply:
you mean just adding `// DEPRECATED`? (or `// Deprecated`? I see both in different files...)

Reply:
Yeah, I doubt we'll be able to ever remove files for stable API /cc @derekwaynecarr but at least we can be clear they are deprecated.